src/components/PopoverMenu/popoverMenuPropTypes.js

```diff
@@ -49,13 +49,17 @@ const propTypes = {
 
     /** Whether disable the animations */
     disableAnimation: PropTypes.bool,
+
+    /** Whether to allow arrow key actions on the list */
+    shouldEnableArrowKeysActions: PropTypes.bool,
```

Comment:
Wondering why we don't just enable this by default and not introduce this prop ... I get that it would make the change in many more places across the app, but standardizing on arrow navigation everywhere seems like a good idea. Due to the extra QA that would be needed we could double the budget.

I would maybe rather do that than have a number of different PRs just to add this prop into different components, and then at some later date just remove the prop entirely.

Reply:
Another advantage of this approach is that we can create a single issue to update the regression suite with all the tests needed in the different places where `<PopoverMenu />` is used

Reply:
I did think about reversing it by making it true by default and we explicitly disable where not needed. There are about 5-6 places where `<PopoverMenu />` is used. 

> I would rather do that than have a number of different PRs just to add this prop into different components
Good thought.

Reply:
I looked though the usage and I _think_ we can have keyboard navigation everywhere this component is used. If that's the case we can just remove this prop entirely

Reply:
Okay give me a day I'll remove the prop, test again for the other screens. 

> Due to the extra QA that would be needed we could double the budget.

It would be great if this can be done.

Reply:
Updated